src/vm/eventpipe.cpp

```diff
@@ -29,17 +29,17 @@
 #ifdef FEATURE_PERFTRACING
 
 CrstStatic EventPipe::s_configCrst;
-Volatile<bool> EventPipe::s_tracingInitialized = false;
+Volatile<bool> EventPipe::s_tracingInitialized(false);
 EventPipeConfiguration EventPipe::s_config;
 EventPipeEventSource *EventPipe::s_pEventSource = nullptr;
 VolatilePtr<EventPipeSession> EventPipe::s_pSessions[MaxNumberOfSessions];
+uint32_t EventPipe::s_NumberOfSessions = 0;
```

Comment:
nit: all the other static variables are prefixed with `s_` followed by a lowercase character. Maybe we should name this `s_numberOfSessions` to follow the trend